### PR TITLE
Fix/restart deposits

### PR DESCRIPTION
**Pull request by FrozenKiwi**
Not technically a fix, as it wasn't implemented.

Same mechanism for e-Transfers applied to Deposits

### CODE REVIEW

File: libs/rbcapi/src/action.ts [hunk 1]
```diff
@@ -59,7 +60,9 @@ export class ApiAction {
   step: number = 0;
 
   private constructor(identifier: string) {
-    this.outCache = `/temp/TheCoin/logs/rbcapi/Screenshots/${identifier}`;
+    const base = process.env.TC_LOG_FOLDER ?? tmpdir()
```
Comment (FrozenKiwi) [inline libs/rbcapi/src/action.ts:63]: hard-coded path broke linux (and all good coding standards)

File: apps/admin/src/containers/BrokerCAD/Clients/ManualOverride.tsx [hunk 2]
```diff
@@ -2,6 +2,7 @@ import React, { useState } from 'react';
 import { AnyAction } from 'redux';
 import { ActionType, storeTransition } from '@thecointech/broker-db';
 import { graph } from '@thecointech/tx-etransfer';
+import { etransfer } from '@thecointech/tx-deposit';
```
Comment (FrozenKiwi) [inline apps/admin/src/containers/BrokerCAD/Clients/ManualOverride.tsx:5]: possibly need better names than 'graph' for the exported transition graphs for our actions

File: apps/admin/src/containers/BrokerCAD/Clients/ManualOverride.tsx [hunk 3]
```diff
@@ -22,10 +23,10 @@ export const ManualOverride = (props: AnyAction) => {
   )
 }
 
-function buildOptions(_type: ActionType) {
+function buildOptions(type: ActionType) {
   // What states can we go to?
-  // Assume eTransfer for now
-  const states = Object.keys(graph);
+  // We may need to consider a cleaner view than the raw to-any-state
+  const states = type == "Buy" ? Object.keys(etransfer) : Object.keys(graph);
```
Comment (FrozenKiwi) [inline apps/admin/src/containers/BrokerCAD/Clients/ManualOverride.tsx:29]: does not consider manual purchases, as they will probably never fail (can implement it if it ever proves necessary)

Comment (MarieTheCoin) [reply apps/admin/src/containers/BrokerCAD/Clients/ManualOverride.tsx:29]: "will never fail" makes me really anxious every times I heard that one it failed at some point

File: apps/admin/src/containers/BrokerCAD/Clients/ManualOverride.tsx (hunk 2 above)
Comment (MarieTheCoin) [reply apps/admin/src/containers/BrokerCAD/Clients/ManualOverride.tsx:5]: Is it linked to the "visual" graph?

**Review by MarieTheCoin — APPROVED** [review]

File: apps/admin/src/containers/BrokerCAD/Clients/ManualOverride.tsx (hunk 3 above)
Comment (FrozenKiwi) [reply apps/admin/src/containers/BrokerCAD/Clients/ManualOverride.tsx:29]: you're right.  Implementing that now

**Pull request merged**